scl/pgsql/pgsql.conf

```diff
@@ -91,37 +91,19 @@ block parser postgresql-csvlog-parser(internal(yes) prefix(".pgsql.") on-type-er
             set(string("$(if ('${`prefix`connection_from}' ne '') ${`prefix`connection_from} $HOST_FROM)") value("`prefix`connection_from") internal(yes));
             set(string("${`prefix`message}") value("MESSAGE") internal(yes));
         };
-        # Map severity
-        if (match("DEBUG[1-5]" value('`prefix`severity') type(pcre))) {
+
+        # Map severity to match https://www.postgresql.org/docs/current/runtime-config-logging.html#RUNTIME-CONFIG-SEVERITY-LEVELS
+	# for some reason PgSQL decided to map its internal levels in a
+        # shifted fashion above and including "error".
```

Comment:
PANIC should be shifted by 2:

| Severity | Usage | Syslog | eventlog |
| --- | --- | --- | --- |
| PANIC | Reports an error that caused all database sessions to abort. | CRIT | ERROR

```c
panic,		SYSLOG_SEVERITY_CODE(0)
crit,		SYSLOG_SEVERITY_CODE(2)
```

Reply:
good catch. thanks. I added a testcase and a fix.